### ISSUE TITLE
Cannot install if O365 is installed

### ISSUE THREAD

**Issue by IzakIsMe** [MicrosoftDocs/OfficeDocs-DeployOffice#198]
I have an environment on which we use O365 Pro Plus.
I setup the XML to install both Project 2019 X86 and O365 Pro plus.
Only O365 installs, Project 2019 cannot install.
If I only install Project 2019 and then a separate install of O365 Pro Plus, I get blocked by a warning.

**Comment by manoth-msft** [comment]
Closing due to the age of the issue report. Feel free to re-open or open new issue if problem persists.
